Code/Framework/AzToolsFramework/AzToolsFramework/FocusMode/FocusModeNotificationBus.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+
+namespace AzToolsFramework
+{
+    //! Used to notify when the editor focus changes.
+    class FocusModeNotifications
+        : public AZ::EBusTraits
```

Comment:
question: I take it this will also be updated to use `AZ::EntityContextId` in future?

Reply:
Done!